src/database/migrations/20211008033154_areas.ts

```diff
@@ -0,0 +1,17 @@
+import { Knex } from 'knex'
+
+export async function up (knex: Knex): Promise<void> {
+  return knex.schema.hasTable('areas').then(function (exists) {
+    if (!exists) {
+      return knex.schema.createTable('areas', function (table) {
+        table.string('id', 4).primary()
+        table.string('name', 60).notNullable()
+        table.boolean('is_active').notNullable()
```

Comment:
This table will be used in cities, districts, and villages. I'm just curious what happens if a village belongs to area A, but its district belongs to area B? Or are they guaranteed to be in the same area?